File: livingdoc-fixture-api/src/main/java/org/livingdoc/fixture/api/converter/time/LocalDateTimeConverter.java
package org.livingdoc.fixture.api.converter.time;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;


public class LocalDateTimeConverter extends AbstractTemporalConverter<LocalDateTime> {

    @Override
    DateTimeFormatter getDefaultFormatter() {
        return DateTimeFormatter.ISO_LOCAL_DATE_TIME;
    }

    @Override
    LocalDateTime parse(String value, DateTimeFormatter formatter) {
        return LocalDateTime.parse(value, formatter);
    }

}

File: livingdoc-fixture-converter/src/main/kotlin/org/livingdoc/fixture/converter/time/LocalDateTimeConverter.kt
package org.livingdoc.fixture.converter.time

import java.time.LocalDateTime
import java.time.format.DateTimeFormatter


open class LocalDateTimeConverter : AbstractTemporalConverter<LocalDateTime>() {
    override fun defaultFormatter(): DateTimeFormatter = DateTimeFormatter.ISO_LOCAL_DATE_TIME
    override fun doParse(value: String, formatter: DateTimeFormatter): LocalDateTime = LocalDateTime.parse(value, formatter)
}
